web/styles/reactions.css

```diff
@@ -147,7 +147,6 @@
     }
 
     .emoji-popover {
-        display: block;
```

Comment:
Similarly with this one -- if you say why it's unnecessary, that is a huge help to a reviewer.

Reply:
I understood the concept, thanks!

This is just a cleanup that should act as a preparatory commit. The `.emoji-popover` div element has a `block` value by default.

Reply:
Added to the commit message.

Reply:
Thanks!